core/query/annotated.go

```diff
@@ -66,6 +67,39 @@ type SpentOutput struct {
 	Position      uint32             `json:"position"`
 }
 
+type AnnotatedAccount struct {
+	ID     string           `json:"id"`
+	Alias  string           `json:"alias,omitempty"`
+	Keys   []*AccountKey    `json:"keys"`
+	Quorum int              `json:"quorum"`
+	Tags   *json.RawMessage `json:"tags"`
+}
+
+type AccountKey struct {
+	RootXPub              chainkd.XPub         `json:"root_xpub"`
+	AccountXPub           chainkd.XPub         `json:"account_xpub"`
+	AccountDerivationPath []chainjson.HexBytes `json:"account_derivation_path"`
+}
+
+type AnnotatedAsset struct {
+	ID              chainjson.HexBytes `json:"id"`
```

Comment:
I think I originally avoided the `bc` types in this package for consistency between fields, so that all of the hex-encoded byte arrays on the annotated structs use the same type. Looking at it again, that might not be helpful/useful/clear. I'll switch em over to using the `bc` types where possible and see how it looks.